src/datepicker/datepicker.component.spec.ts

```diff
@@ -0,0 +1,119 @@
+import { FormsModule } from "@angular/forms";
+
+@Component({
+	template: `
+	<ibm-date-picker
+		[label]="label"
+		[placeholder]="placeholder"
+		[theme]="theme"
+		[value]="value"
+		[disabled]="disabled"
+		[invalid]="invalid"
+		[invalidText]="invalidText"
+		[dateFormat]="dateFormat"
+		(valueChange)="valueChange($event)">
+	</ibm-date-picker>
+	`
+})
+class DatePickerTest {
+	label = "Date picker label";
+	placeholder = "mm/dd/yyyy";
+	theme = "dark";
+	value = ["10/19/2019"];
+	disabled = false;
+	dateFormat = "m/d/Y";
+	invalid = false;
+	valueChange(event) {
+		console.log(event);
+	}
+}
+
+describe("DatePicker", () => {
+	let fixture, element;
+	beforeEach(() => {
+		TestBed.configureTestingModule({
+			declarations: [
+				DatePicker,
+				DatePickerTest,
+				DatePickerInput
+			],
+			imports: [
+				Calendar16Module,
+				FormsModule
+			]
+		});
+	});
+
+	it("should work", () => {
+		fixture = TestBed.createComponent(DatePicker);
+		expect(fixture.componentInstance instanceof DatePicker).toBe(true);
+	});
+
+	it("should set label to Date picker label", () => {
+		fixture = TestBed.createComponent(DatePickerTest);
+		element = fixture.debugElement.query(By.css("ibm-date-picker"));
+		fixture.detectChanges();
+		expect(element.nativeElement.textContent).toEqual("Date picker label");
+	});
+
+	it("should set input value to 10/19/2019", () => {
+		fixture = TestBed.createComponent(DatePickerTest);
+		fixture.detectChanges();
+		element = fixture.debugElement.query(By.css("input"));
+		expect(element.nativeElement.value).toEqual("10/19/2019");
+	});
+
+	it("should set placeholder to mm/dd/yyyy", () => {
+		fixture = TestBed.createComponent(DatePickerTest);
+		fixture.detectChanges();
+		element = fixture.debugElement.query(By.css("input"));
+		expect(element.nativeElement.placeholder).toEqual("mm/dd/yyyy");
+	});
+
+	it("should disable input", () => {
+		fixture = TestBed.createComponent(DatePickerTest);
+		fixture.componentInstance.disabled = true;
+		fixture.detectChanges();
+		element = fixture.debugElement.query(By.css("input"));
+		expect(element.nativeElement.disabled).toBe(true);
+	});
+
+	it("should set date format to m/d/Y", () => {
+		fixture = TestBed.createComponent(DatePickerTest);
+		element = fixture.debugElement.query(By.css("ibm-date-picker"));
+		fixture.detectChanges();
+		expect(element.componentInstance.dateFormat).toEqual("m/d/Y");
+	});
+
+	it("should set theme to dark", () => {
+		fixture = TestBed.createComponent(DatePickerTest);
+		element = fixture.debugElement.query(By.css("ibm-date-picker"));
+		fixture.detectChanges();
+		expect(element.componentInstance.theme).toEqual("dark");
+	});
+
+	it("should set invalid to true", () => {
+		fixture = TestBed.createComponent(DatePickerTest);
+		element = fixture.debugElement.query(By.css("ibm-date-picker"));
+		fixture.componentInstance.invalid = true;
+		fixture.detectChanges();
+		expect(element.componentInstance.invalid).toBe(true);
```

Comment:
Is there some other change we can look for? An additional element or class?

Reply:
I can't seen to find anything else besides ng-reflect attributes.